cli/tests/e2e/test_utility_commands.py

```diff
@@ -0,0 +1,53 @@
+import re
+import subprocess
+
+import pytest
+from tests.semgrep_runner import SEMGREP_BASE_COMMAND
+
+
+@pytest.mark.kinda_slow
```

Comment:
is this kinda slow?

Reply:
well. it's from the old code, but still

Reply:
Anything that invokes semgrep is kind of slow I think (it takes half a second of what I assume is startup time)